build/build-dt-report-resources.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const browserify = require('browserify');
```

Comment:
Perhaps gulp would do better here?

Reply:
nope.

Reply:
Just of curiousity, why are you using browserify?

Reply:
> Just of curiosity, why are you using browserify?

we need to run the same files in node and the browser. Even if we ran a different bundler, we'd probably run the same browserify transforms, so, might as well run the real deal until such a time something better comes along :)

Reply:
Gotcha. Thanks for sharing